### PR TITLE
Use gradlew to execute build or clean

### PR DESCRIPTION
**Pull request by winksaville**
Updated README.md adding a Potential Issus section for Gradle and updated
the build and clean commands.

### CODE REVIEW

**Comment by winksaville** [comment]
Testing solving issue #4 with Gradle. The primary reason to use gradle is that on some systems, such as Arch Linux, gradle fails because of termcap issues. Bu using it for PerspectiveSuite it will fail faster and be more obvious.

File: README.md [hunk 1]
```diff
@@ -44,7 +58,7 @@ This only cleans the gradle based builds as other
 builds are currently unconditionally built.
 
 ```
-$ ./clean.sh
+$ .gradlew clean
```
Comment (stuartmscott) [inline README.md]: ./gradlew

**Comment by winksaville** [comment]
I did my typical where I used `git push --force`, I could have added as a second commit then when merging we could choose to "Squash and merge" although I wonder if that would actually do a "Squash, Rebase and Merge". Do you have a preference on what you want to do?

Personally, if the "Squash and Merge" ends up to be a "merge commit" then I suggest we use the `git push --force" technique, but I don't know how it works, do you?

File: README.md (hunk 1 above)
Comment (winksaville) [reply README.md]: fixed

**Comment by stuartmscott** [comment]
My typical work flow is to amend to the commit and then force push to the pull request branch, as you have done, rather than squashing the commits at merge time.

**Review by stuartmscott — APPROVED** [review]

**Comment by stuartmscott** [comment]
'./gradew build' works for me!

One small issue is that the "build" and "clean" tasks don't show up in './gradlew tasks';

```
$ ./gradlew tasks

> Task :tasks

------------------------------------------------------------
Tasks runnable from root project
------------------------------------------------------------

Build Setup tasks
-----------------
init - Initializes a new Gradle build.
wrapper - Generates Gradle wrapper files.

Help tasks
----------
buildEnvironment - Displays all buildscript dependencies declared in root project 'PerspectiveSuite'.
components - Displays the components produced by root project 'PerspectiveSuite'. [incubating]
dependencies - Displays all dependencies declared in root project 'PerspectiveSuite'.
dependencyInsight - Displays the insight into a specific dependency in root project 'PerspectiveSuite'.
dependentComponents - Displays the dependent components of components in root project 'PerspectiveSuite'. [incubating]
help - Displays a help message.
model - Displays the configuration model of root project 'PerspectiveSuite'. [incubating]
projects - Displays the sub-projects of root project 'PerspectiveSuite'.
properties - Displays the properties of root project 'PerspectiveSuite'.
tasks - Displays the tasks runnable from root project 'PerspectiveSuite'.

To see all tasks and more detail, run gradlew tasks --all

To see more detail about a task, run gradlew help --task <task>

BUILD SUCCESSFUL in 649ms
1 actionable task: 1 executed
```

They do show up in './gradlew tasks --all' albeit at the bottom under "Other tasks";

```
$ ./gradlew tasks --all

> Task :tasks

------------------------------------------------------------
Tasks runnable from root project
------------------------------------------------------------

Build Setup tasks
-----------------
init - Initializes a new Gradle build.
wrapper - Generates Gradle wrapper files.

Help tasks
----------
buildEnvironment - Displays all buildscript dependencies declared in root project 'PerspectiveSuite'.
components - Displays the components produced by root project 'PerspectiveSuite'. [incubating]
dependencies - Displays all dependencies declared in root project 'PerspectiveSuite'.
dependencyInsight - Displays the insight into a specific dependency in root project 'PerspectiveSuite'.
dependentComponents - Displays the dependent components of components in root project 'PerspectiveSuite'. [incubating]
help - Displays a help message.
model - Displays the configuration model of root project 'PerspectiveSuite'. [incubating]
projects - Displays the sub-projects of root project 'PerspectiveSuite'.
properties - Displays the properties of root project 'PerspectiveSuite'.
tasks - Displays the tasks runnable from root project 'PerspectiveSuite'.

Other tasks
-----------
build
clean
prepareKotlinBuildScriptModel

BUILD SUCCESSFUL in 781ms
1 actionable task: 1 executed
```

**Comment by winksaville** [comment]
Any idea how to fix that?

**Comment by winksaville** [comment]
> Any idea how to fix that?

And answer seems to be I need to add it to a [group](https://guides.gradle.org/writing-gradle-tasks/#add_a_task_description):

"Also (and since Gradle 3.3) if a task does not have a group it will not be listed unless --all is specified."

I'll do that

**Pull request merged**